content/modules/ROOT/pages/self-service-patterns/lab-namespace-self-service-patterns.adoc

```diff
@@ -1,175 +1,159 @@
+During this lab, you will have the opportunity to explore different project customizations across different clusters.
 
 
 [#lab]
 == Scenario
-Now, imagine for a moment that you need to fullfil a need for the Python Development team. The Python Team's responsibility is to create a new start prototyping with AI and for this you need a namespace, this new namespace is not part of the Software Development Lifecycle. The request is having a namespace isolated from the rest of the cluster, with resource limits but enough resources to experiment.
+Now, imagine for a moment that you need to fulfill a need for the Python Development team. The Python Team needs the ability to create new namespaces to start prototyping with AI, and these new namespaces are not part of the Software Development Lifecycle. These new namespaces should be isolated from the rest of the cluster, with resource limits but sufficient resources for experimentation.
 
 
 == Explore the Project template in GitLab
-. Take the time to review the source code to get familiarize with the content.
-
-
-* {gitlab_url}/rhdh/rhads-enablement-l3-st-self-service/-/blob/main/namespace-medium[{gitlab_url}/rhdh/rhads-enablement-l3-st-self-service/-/blob/main/namespace-medium,window=_blank]
-
-** Login with your user credentials:
+. Take the time to {gitlab_url}/rhdh/rhads-enablement-l3-st-self-service/-/blob/main/namespace-medium[review existing template^] in GitLab to get familiar with the content. If you need to log in, use your GitLab credentials:
 
     ** *Username*: {gitlab_user}
     ** *Password*: {gitlab_user_password}
 
+Open the following files or folders to examine their contents:
 
-* Component/catalog-info.yaml: This file will represent the object in Developer Hub, such as name and links.
-+
-image:self-service-patterns/namespace-lab/source-code-catalog-info.png[Project Catalog]
-
-* Developer Hub, powered by GitOps, maintains the desired state as the actual state in the cluster for any template definitions, ensuring system stability.
-+
-image:self-service-patterns/namespace-lab/source-code-manifests.png[Project Manifests]
+* `template.yaml`: This file defines the UI experience for the end user in Developer Hub (e.g. the fields and drop-downs and selectors to define the configuration of the namespace in this case) and the programmatic steps to obtain software templates, create repositories, create GitOps objects, and more.
 
-* GitOps application definitions and necessary secrets to read from the SCM. In this case, the argo-app-dev.yaml is the Argo CD Application pointing to the Namespace manifest's folder.
+* `skeleton/catalog-info.yaml`: This file represents the object that is added to the catalog in Developer Hub when this template is used, including its name and links. All of the placeholders (e.g. `${{values.component_id}}`) are replaced with actual values user enters or are dynamically computed when the template is used.
```

Comment:
replace with:

`skeleton/catalog-info.yaml`: This file represents a descriptor file used to define and register entities and will contain information displayed in the component's overview, such as source code, technical docs links. All of the placeholders (e.g., `${{values.component_id}}`) are replaced with actual values the user enters or are dynamically computed when the template is used.